packages/ocs/dashboards/persistent-internal/capacity-trend-card/capacity-trend-card.tsx

```diff
@@ -115,28 +144,57 @@ const CapacityTrendCard: React.FC = () => {
     basePath: usePrometheusBasePath(),
   });
 
-  const loadError = availableCapacityError || totalUtilError || uptimeError;
+  const [totalCapacity, totalCapacityError, totalCapacityLoading] =
+    useCustomPrometheusPoll({
+      query:
+        CEPH_CAPACITY_BREAKDOWN_QUERIES[
+          StorageDashboardQuery.CEPH_CAPACITY_TOTAL
+        ],
+      endpoint: 'api/v1/query' as any,
+      basePath: usePrometheusBasePath(),
+    });
+
+  const loadError =
+    availableCapacityError ||
+    totalUtilError ||
+    uptimeError ||
+    totalCapacityError;
   const loading =
-    availableCapacityLoading || totalUtilLoading || uptimeUtilLoading;
+    availableCapacityLoading ||
+    totalUtilLoading ||
+    uptimeUtilLoading ||
+    totalCapacityLoading;
 
   let daysUp: number;
   if (!loading && !loadError) {
     daysUp = calculateDaysUp(parser(uptime));
   }
 
   const totalUtilMetric = parser(totalUtil);
+  const totalCapacityMetric = parser(totalCapacity);
   const avgUtilMetric =
     !!daysUp && !!totalUtilMetric
       ? totalUtilMetric / Math.ceil(daysUp) // do a ceil function, if it's a new cluster (dayUp < 1)
       : 0;
-  const avgUtilMetricByte = humanizeBinaryBytes(avgUtilMetric.toFixed(0));
 
+  const avgUtilMetricByte = humanizeBinaryBytesWithNegatives(
+    avgUtilMetric.toFixed(0)
+  );
   const availableCapacityMetric = parser(availableCapacity);
-  const daysLeft =
+  let daysLeft =
     !!availableCapacityMetric && avgUtilMetric
       ? Math.floor(availableCapacityMetric / avgUtilMetric)
       : 0;
 
+  if (daysLeft < 0) {
+    // Clean up negative average values, if a user empties a cluster the average value goes to negative we need to make
+    // sure that estimated days left takes into account the average rate of filling up the cluster
+    const cluster_clean_up_to_zero_days =
+      totalUtilMetric / Math.abs(avgUtilMetric);
+    const cluster_fill_up_to_max_days =
```

Comment:
I comment using `_` case for reference... we don't follow that in repo...
```suggestion
    const clusterCleanUpToZeroDays =
      totalUtilMetric / Math.abs(avgUtilMetric);
    const clusterFillUpToMaxDays =
```

Reply:
oops, fixing